homeassistant/components/discord/notify.py

```diff
@@ -33,6 +33,15 @@ def __init__(self, hass, token):
         self.token = token
         self.hass = hass
 
+    def file_exists(self, filename):
+        """Check if a file exists on disk and is in authorized path."""
+        import os.path
```

Comment:
Standard library imports can be put at the top of the module.

Reply:
Done !
Should I squash the review commits for a cleaner history ?

Reply:
No, please don't squash commits after review has started to make it easier for readers to track changes. We will squash when merging.